packages/node-cli/src/commands/actionresult/ManageAccountActionResult.js

```diff
@@ -0,0 +1,73 @@
+/*
+ ** Copyright (c) 2020 Oracle and/or its affiliates.  All rights reserved.
+ ** Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+ */
+'use strict';
+const assert = require('assert');
+const { ActionResult, ActionResultBuilder } = require('./ActionResult');
+
+const STATUS = {
+	ERROR: 'ERROR',
+	SUCCESS: 'SUCCESS',
+};
+
+const MANAGE_ACTION = {
+	LIST: 'list',
+	EXIT: 'exit',
```

Comment:
I don't thinks exit is a manage_action

Reply:
Then, it should not be called MANAGE_ACTION

Reply:
We close it for now... since this is already delaying the story